recipes/boost/meta.yaml

```diff
@@ -0,0 +1,119 @@
+    - test -f $PREFIX/lib/libboost_signals.dylib              # [osx]
+    - test -f $PREFIX/lib/libboost_system.dylib               # [osx]
+    - test -f $PREFIX/lib/libboost_thread.dylib               # [osx]
+    - test -f $PREFIX/lib/libboost_timer.dylib                # [osx]
+    - test -f $PREFIX/lib/libboost_type_erasure.dylib         # [osx]
+    - test -f $PREFIX/lib/libboost_unit_test_framework.dylib  # [osx]
+    - test -f $PREFIX/lib/libboost_wave.dylib                 # [osx]
+    - test -f $PREFIX/lib/libboost_wserialization.dylib       # [osx]
+
+    # Verify shared object.
+    - test -f $PREFIX/lib/libboost_atomic.so                  # [linux]
+    - test -f $PREFIX/lib/libboost_chrono.so                  # [linux]
+    - test -f $PREFIX/lib/libboost_container.so               # [linux]
+    - test -f $PREFIX/lib/libboost_context.so                 # [linux]
+    - test -f $PREFIX/lib/libboost_coroutine.so               # [linux]
+    - test -f $PREFIX/lib/libboost_date_time.so               # [linux]
+    - test -f $PREFIX/lib/libboost_filesystem.so              # [linux]
+    - test -f $PREFIX/lib/libboost_graph.so                   # [linux]
+    - test -f $PREFIX/lib/libboost_iostreams.so               # [linux]
+    - test -f $PREFIX/lib/libboost_locale.so                  # [linux]
+    - test -f $PREFIX/lib/libboost_log.so                     # [linux]
+    - test -f $PREFIX/lib/libboost_log_setup.so               # [linux]
+    - test -f $PREFIX/lib/libboost_math_c99.so                # [linux]
+    - test -f $PREFIX/lib/libboost_math_c99f.so               # [linux]
+    - test -f $PREFIX/lib/libboost_math_c99l.so               # [linux]
+    - test -f $PREFIX/lib/libboost_math_tr1.so                # [linux]
+    - test -f $PREFIX/lib/libboost_math_tr1f.so               # [linux]
+    - test -f $PREFIX/lib/libboost_math_tr1l.so               # [linux]
+    - test -f $PREFIX/lib/libboost_prg_exec_monitor.so        # [linux]
+    - test -f $PREFIX/lib/libboost_program_options.so         # [linux]
+    - test -f $PREFIX/lib/libboost_python.so                  # [linux]
+    - test -f $PREFIX/lib/libboost_random.so                  # [linux]
+    - test -f $PREFIX/lib/libboost_regex.so                   # [linux]
+    - test -f $PREFIX/lib/libboost_serialization.so           # [linux]
+    - test -f $PREFIX/lib/libboost_signals.so                 # [linux]
+    - test -f $PREFIX/lib/libboost_system.so                  # [linux]
+    - test -f $PREFIX/lib/libboost_thread.so                  # [linux]
+    - test -f $PREFIX/lib/libboost_timer.so                   # [linux]
+    - test -f $PREFIX/lib/libboost_type_erasure.so            # [linux]
+    - test -f $PREFIX/lib/libboost_unit_test_framework.so     # [linux]
+    - test -f $PREFIX/lib/libboost_wave.so                    # [linux]
+    - test -f $PREFIX/lib/libboost_wserialization.so          # [linux]
```

Comment:
This is pretty gross as it is soooo long. Sorry, many libraries. However, this makes it trivial to add more when they show up and it is clear to anyone what is going on here.

Only have done Mac and Linux. If someone can point me to the equivalent syntax for Windows, I can add that now to.